hack/e2e/kops.sh

```diff
@@ -57,26 +52,27 @@ function kops_create_cluster() {
     kops_patch_cluster_file "$CLUSTER_FILE" "$KOPS_PATCH_FILE"
 
     loudecho "Creating cluster $CLUSTER_NAME with $CLUSTER_FILE"
-    ${KOPS_BIN} create --state "${KOPS_STATE_FILE}" -f "${CLUSTER_FILE}"
+    ${BIN} create --state "${KOPS_STATE_FILE}" -f "${CLUSTER_FILE}"
+    kops create secret --state "${KOPS_STATE_FILE}" --name "${CLUSTER_NAME}" sshpublickey admin -i "${SSH_KEY_PATH}".pub
```

Comment:
this is unrelated change, cuz I found out while testing "kops update cluster --state "${KOPS_STATE_FILE}" "${CLUSTER_NAME}" \
    --ssh-public-key="${SSH_KEY_PATH}".pub --yes" is deprecated:

--ssh-public-key string         SSH public key to use (deprecated: use kops create secret instead)